packages/Molecules/Search/SearchForm.vue

```diff
@@ -145,6 +146,7 @@ export default {
       @include rpl_typography_ruleset($rpl-search-form-input-ruleset);
       background-color: transparent;
       border: 0;
+      padding: 1px;
```

Comment:
This 1px is to match Chrome's default - it provides a little padding so when the focus indicator is on the field, the indicator isn't right up against the text.